src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+body {
+  width: 100%;
+  background: #e5e5e5;
+  margin: 0;
+}
+
+.header {
+  box-sizing: border-box;
```

Comment:
all other of your elements don't have box-sizing: border-box. It means that width property is calculated differently for them. Move it to * selector